DMCompiler/Compiler/DM/DMAST.cs

```diff
@@ -1329,10 +1329,17 @@ public sealed class DMASTList : DMASTExpression {
         }
 
         public bool AllValuesConstant() {
-            return Values.All(value => value is {
-                Key: DMASTExpressionConstant,
-                Value: DMASTExpressionConstant
-            });
+            return Values.All(
+                value => (value is {
+                    Key: DMASTExpressionConstant,
+                    Value: DMASTExpressionConstant
+                })
+                ||
+                (value is {
+                    Key: DMASTExpressionConstant,
+                    Value: DMASTList
+                } && ((DMASTList) value.Value).AllValuesConstant())
```

Comment:
Could be cleaned up a bit by removing the cast
```suggestion
                    Value: DMASTList valueList
                } && valueList.AllValuesConstant())
```